### DISCUSSION TITLE
Move to New Domain

### DISCUSSION THREAD

**Discussion by faridalizade** [FreshRSS/FreshRSS#4935] [Q&A] [answered]
Hello,

I want to duplicate my old FreshRSS from 1domain.com to 2domain.com.. 

I did it but it seems 500 error appears. 

What am I missed?

Thank you.

**Comment by UPuXA** [comment]
Hello
i think you need to change a config file in the main (not p) FreshRSS directory.

I changed the root of my apache (domain.net/p -> domain.net/)  and had to change this . The setting should be near to the top and the file ist called config something.

**Comment by Alkarex** [comment, marked as answer]
As @UPuXA wrote, after moving, edit your `./FreshRSS/data/config.php` to change the `base_url` address.
In case of error, check your FreshRSS logs in `./FreshRSS/data/users/*/log*.txt` and Web server logs.

**Comment by faridalizade** [comment]
Hello,

I did above mentioned and an error has gone when I changed server name, db username.

Fixed. Thanks
